dags/xcom_gcs_ds.py

```diff
@@ -33,6 +23,8 @@ def using_gcs_for_xcom_ds():
 
     @task
     def load_data():
+        """Pull Census data from Public BigQuery and save as Pandas dataframe in GCS bucket with XCom"""
+
         client = bigquery.Client()
```

Comment:
https://github.com/apache/airflow/blob/main/airflow/providers/google/cloud/hooks/bigquery.py#L204

Can we use the hook here with the get df method? It'll simplify the `requirements.txt` as well to only need the Google provider

Reply:
Ohhh...did not know that hook existed! Yes, will update.